nais-dev-gcp.yaml

```diff
@@ -33,3 +33,5 @@ spec:
     value: dfafewsfjeskafejsafeskalfjesaklfjeklasjfklsefjlsalkfekljasfjksejklfasekljfkljaskle
   - name: NEXT_PUBLIC_BASE_PATH
     value: /syk/dinesykmeldte
+  - name: NEXT_PUBLIC_SENTRY_ENV
+    value: dev
```

Comment:
Denne kunne vært generell uten 'SENTRY' i navnet

Reply:
eller, maybe not? sentryconfigen forventer kanskje en av et sett predefined strings her?